src/asm/symbol.c

```diff
@@ -222,6 +222,21 @@ static void fullSymbolName(char *output, size_t outputSize,
 		fatalerror("Symbol name is too long: '%s%s'\n", scopeName, localName);
 }
 
+static void assignStringSymbol(struct Symbol *sym, char const *value)
+{
+	size_t len = strlen(value);
+	char *string = malloc(len + 1);
+
+	if (string == NULL)
+		fatalerror("No memory for string equate: %s\n", strerror(errno));
+	strcpy(string, value);
```

Comment:
Isn't all that just `strdup`?

Reply:
Now that you mention it, yes. I'd just moved that code out of `sym_AddString`.

Reply:
Simplified to `strdup`.